### PR TITLE
Add more intuitive arguments to default.launch

### PR DESCRIPTION
**Pull request by egordon**
(1) Add all controllers from https://github.com/personalrobotics/pr_ros_controllers/pull/26 to YAML file for use.

(2) Modify default.launch to be sensitive to arm version and DOF. In preparation for making the repo more robot-agnostic.

Note: this should not change any functionality as-is. It is mostly a refactor in preparation for future PRs.

***

**Before creating a pull request**

- [ N/A ] Document new methods and classes
- [ N/A ] Format code with `make format`

**Before merging a pull request**

- [ N/A ] Add unit test(s) for this change

### CODE REVIEW

File: launch/default.launch [hunk 1]
```diff
@@ -1,31 +1,35 @@
 <launch>
 
-  <arg name="feeding" default="false" doc="Boolean flag of whether to set up the robot for the feeding demo"/>
-  <arg name="perception" default="true" doc="Boolean flag of whether to set up the real perception for the feeding demo"/>
+  <arg name="use_forque" default="false" doc="Add forque to URDF and spawned controllers."/>
+  <arg name="perception" default="false" doc="Boolean flag of whether to set up the perception on ADA" />
   <!-- Other options: retinanet, spnet, spanet -->
   <!-- See possible networks in https://github.com/personalrobotics/food_detector/ -->
-  <arg name="detector" default="retinanet" doc="Which network to use for food perception."/>
+  <arg name="detector" default="false" doc="Which network to use for food perception. Set to 'false' for simulation."/>
```
Comment (amalnanavati) [inline launch/default.launch:7]: Are the options other than 'false' documented anywhere? If not, perhaps it should be in the docstring here?

File: launch/default.launch [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
    <node pkg="jaco_hardware"
           name="ros_control_kinova_jaco"
           type="jaco_hardware"
-          output="screen" />
+          output="screen"
+          if="$(eval arg('version') == 2)" />
 
    <!-- load controllers -->
-   <rosparam file="$(find libada)/config/controllers.yaml" command="load" />
+   <rosparam file="$(find libada)/config/gen$(arg version)_$(arg dof)dof.yaml" command="load" />
```
Comment (amalnanavati) [inline launch/default.launch:19]: Just out of curiosity, would the launchfile fail if this file is not found, or would the error be caught and the launchfile continue without these parameters? (Desired functionality imo is the launchfile failing at this line with a "file not found" error so users realize their mistake)

**Review by amalnanavati — APPROVED** [review]
LGTM. I'm approving because the comments are either minor or just clarifying comments. If you end up addressing the minor comment I'll re-approve, but I also think it's fine to merge it in as-is.

File: launch/default.launch (hunk 2 above)
Comment (egordon) [reply launch/default.launch:19]: Looks like this is the behavior, so we're good here

https://answers.ros.org/question/236002/roslaunch-stop-if-yaml-doesnt-exists/

File: launch/default.launch (hunk 1 above)
Comment (egordon) [reply launch/default.launch:7]: There is, see the preceding 2 lines. I could make it more explicit

File: launch/default.launch (hunk 2 above)
Comment (amalnanavati) [reply launch/default.launch:19]: Awesome! (I didn't see it documented in the docs, but I didn't think to look in the forums -- good catch!)

File: launch/default.launch (hunk 1 above)
Comment (amalnanavati) [reply launch/default.launch:7]: Ah, I must have missed that. No wonder I was feeling deja vu :)

**Pull request merged**